### PR TITLE
Refactor Undo ignored when pasting more than 1 paragraph

### PR DESCRIPTION
**Pull request by santiagoboterokommit**
## Context
Refactor to #144 

## Objective
- [x] Change nested `else if` to function

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -308,6 +308,33 @@ export default class Undo {
...
+      true,
+    );
+  }
+
+  /**
+   * Insert a block when is skipped
+   * @param {Number} prevStateLength is the previous state according to this.position.
+   * @param {Array} state is the current state according to this.position.
+   * @returns 
+   */
```
Comment (miguelnietoa) [inline src/index.js:331]: We could remove `@returns` since this is a void function

File: src/index.js [hunk 2]
```diff
@@ -308,6 +308,33 @@ export default class Undo {
...
+  insertSkippedBlocks(prevStateLength, state) {
+    for (let i = prevStateLength; i < state.length; i += 1) {
```
Comment (miguelnietoa) [inline src/index.js:333]:
```suggestion
    for (let i = prevStateLength; i < state.length; i++) {
```

**Review by miguelnietoa — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (santiagoboterokommit) [reply src/index.js:332]: it is because of the eslint rule

Comment (miguelnietoa) [reply src/index.js:332]: Got it, lets's just keep it so

**Review by miguelnietoa — APPROVED** [review]

**Pull request merged**